frontend/src/api/k8s/notebooks.ts

```diff
@@ -69,6 +79,10 @@ const assembleNotebook = (
     volumeMounts = volumeMounts || [];
     volumeMounts.push(getElyraVolumeMount());
   }
+  volumes = volumes || [];
+  volumes.push(getshmVolume());
+  volumeMounts = volumeMounts || [];
+  volumeMounts.push(getshmVolumeMount());
```

Comment:
This is problematic, when I update the notebook storage, it will duplicated add the memory volumes and volumeMounts

Reply:
Nice catch!

Reply:
So I guess the Elyra ones could have the same problem?

Reply:
cc @harshad16

Reply:
I will try to fix it,
if you have any suggestions on how to fix it, please do share.

Reply:
Made the changes as suggested.

Thanks @DaoDaoNoCode :100: